### PR TITLE
style: new vcent ui

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-1564

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx [hunk 1]
```diff
@@ -1,76 +1,51 @@
...
-        borderRadius: 25,
-        justifyContent: 'space-between',
+        borderRadius: 8,
     },
-
-    VCent: {
-        marginLeft: -10,
-        marginTop: 5,
+    twitter: {
+        fill: '#1D9BF0',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx]: No dead code color. Use a theme color.

File: packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx [hunk 2]
```diff
@@ -80,26 +55,40 @@ export default function VCentDialog({ tweetAddress }: { tweetAddress: string })
     const { value: tweets } = useAsync(() => PluginVCentRPC.getTweetData(tweetAddress), [tweetAddress])
     const tweet = first(tweets)
     usePluginWrapper(tweet?.type === 'Offer')
+    const networkDescriptor = useNetworkDescriptor(NetworkPluginID.PLUGIN_EVM, ChainId.Mainnet)
```
Comment (guanbinrui) [inline packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx]: You can always use a `<PluginIDContextProvider />` and `<PluginWeb3ContextProvider />` if the plugin serves for a specific network. Please take some time to learn it: https://docs.mask.io/docs/web3/hooks.

File: packages/mask/src/plugins/VCent/SNSAdaptor/index.tsx [hunk 3]
```diff
@@ -17,9 +17,18 @@ const sns: Plugin.SNSAdaptor.Definition = {
             name: <Trans i18nKey="plugin_valuables_name" />,
             marketListSortingPriority: 10,
             tutorialLink: 'https://realmasknetwork.notion.site/27424923ee454a4a9b0ed16fc5cb93d0',
-            icon: <Icons.Valuables size={36} />,
+            icon: <Icons.Valuables size={36} style={{ fill: 'white' }} />,
```
Comment (guanbinrui) [inline packages/mask/src/plugins/VCent/SNSAdaptor/index.tsx]: No dead code color.

File: packages/mask/src/plugins/VCent/SNSAdaptor/index.tsx [hunk 4]
```diff
@@ -17,9 +17,18 @@ const sns: Plugin.SNSAdaptor.Definition = {
...
         },
     ],
+    wrapperProps: {
+        icon: (
+            <Icons.Valuables
+                size={24}
+                style={{ filter: 'drop-shadow(0px 6px 12px rgba(0, 0, 0, 0.2))', fill: 'black' }}
```
Comment (guanbinrui) [inline packages/mask/src/plugins/VCent/SNSAdaptor/index.tsx]: No dead code color.

File: packages/plugin-infra/src/types.ts [hunk 5]
```diff
@@ -578,7 +578,7 @@ export namespace Plugin.SNSAdaptor {
 
     export interface PluginWrapperProps {
         icon?: React.ReactNode
-        title?: string
+        title?: string | React.ReactNode | React.ReactElement
```
Comment (guanbinrui) [inline packages/plugin-infra/src/types.ts]:
```suggestion
        title?: string | React.ReactNode
```

File: packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx [hunk 6]
```diff
@@ -80,26 +53,40 @@ export default function VCentDialog({ tweetAddress }: { tweetAddress: string })
...
-                <div className={classes.VCent}>{isDarkTheme() ? <VCentIconDark /> : <VCentIconLight />}</div>
-                <div className={classes.bidInfo}>
-                    <div className={classes.text}>{t('plugin_vcent_last_offer_at')}</div>
-                    <div className={classes.textUSD}> ${tweet.amount_usd}</div>
-                    <div className={classes.textETH}>
-                        (<ETHIcon />
-                        {tweet.amount_eth.toFixed(4)})
-                    </div>
-                </div>
+                href={VALUABLES_VCENT_URL + tweet.tweet_id}>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx]: `urlcat`

File: packages/theme/src/Theme/colors.ts [hunk 7]
```diff
@@ -43,6 +43,7 @@ export const MaskColors = {
             publicSecond: '#767F8D',
             publicThird: '#ACB4C1',
             publicLine: '#F2F5F6',
+            publicTwitter: '#1D9BF0',
```
Comment (guanbinrui) [inline packages/theme/src/Theme/colors.ts:46]: @nuanyang233

File: packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx [hunk 8]
```diff
@@ -1,76 +1,49 @@
...
 const useStyle = makeStyles()((theme) => ({
     root: {
-        marginTop: 20,
-    },
-    content: {
+        margin: theme.spacing(1.5),
         display: 'flex',
-        height: 45,
+        gap: theme.spacing(1),
+        backgroundColor: new Color(theme.palette.maskColor.white).alpha(0.8).toString(),
```
Comment (nuanyang233) [inline packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx]: I recommend `alpha` from `@mui/material`.

eg:
```
import { alpha } from '@mui/material'

...
backgroundColor: alpha(theme.palette.maskColor.white, 0.8)
```

File: packages/theme/src/Theme/colors.ts (hunk 7 above)
Comment (nuanyang233) [reply packages/theme/src/Theme/colors.ts:46]: This color does not exist in the ui kit, please check it.

Comment (lelenei) [reply packages/theme/src/Theme/colors.ts:46]: image

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**